src/neynar-api/neynar-api-client.ts

```diff
@@ -1243,18 +1244,59 @@ export class NeynarAPIClient {
    *
    * @example
    * // Example: Search for users with a specific query
-   * client.searchUser('ris', 19960).then(response => {
+   * client.searchUser('ris', 19960, { limit: 10 }).then(response => {
    *   console.log('User Search Results:', response); // Outputs the results of the user search
    * });
    *
    * For more information, refer to the [Neynar documentation](https://docs.neynar.com/reference/user-search).
    */
   public async searchUser(
     q: string,
-    viewerFid: number
+    viewerFid: number,
```

Comment:
@MrKevinOConnell `viewerFid` is optional now, but please do not make it a part of KV args (it'll be a breaking change).

For now, make it optional, we'll address these breaking changes in the next major release.

Reply:
sounds good!